db/contacts.json

```diff
@@ -0,0 +1,94 @@
+{
+  "contacts": [
+    {
```

Comment:
This should be an object of objects with keys for each contact (just like we will be getting back from Firebase in the real world).